src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
Very true, the best approach to this would be to have the following declaration:
```php
	/**
	 * Static Singleton Holder
	 *
	 * @var self
	 */
	private static $instance;

	/**
	 * Static Singleton Factory Method
	 *
	 * @return self
	 */
	public static function instance() {
		return null === self::$instance ? self::$instance = new self : self::$instance;
	}
```